pom.xml

```diff
@@ -733,6 +734,11 @@
               -Djava.net.preferIPv4Stack=true
               -Djava.awt.headless=true
               -XX:+CMSClassUnloadingEnabled -ea
+              -XX:+IgnoreUnrecognizedVMOptions
+              "--add-modules java.se"
+              -Djdk.attach.allowAttachSelf=true
+              -Duser.language=en
```

Comment:
Why is this necessary?

Reply:
-Djdk.attach.allowAttachSelf=true - for jmockit
-Duser.language=en - format tests fails on non-english system locale
-XX:+IgnoreUnrecognizedVMOptions  "--add-modules java.se" - revalidate it tomorrow

Reply:
http://openjdk.java.net/jeps/261 - java.se module available by default  for unnamed modules, so `-XX:+IgnoreUnrecognizedVMOptions "--add-modules java.se" ` can be removed

Reply:
@oleg-zinovev, since this is another bug, please log a Jira and remove changes connected with locale from this PR.